### PR TITLE
ibm5170.xml: 22 new software additions

### PR DESCRIPTION
**Pull request by ArcadeShadow**
New working software list additions
-----------------------------------
Classics Arcade [ibmpc5150, archive.org]
GrandMaster Chess [The Good Old Days]
GrandMaster Chess Deluxe [The Good Old Days]
Nomad [MarkHnsn, archive.org]
PC Format (Issue 24 - September 1993) [MarkHnsn, archive.org]
PC Format (Issue 26 - November 1993) [MarkHnsn, archive.org]
PC Format (Issue 35 - August 1994) [StefanNL, archive.org]
PC Format (Issue 48 - September 1995) [MarkHnsn, archive.org]
PC Gamer (UK) (Vol. 01 Nº 01 - December 1993) [MarkHnsn, archive.org]
PC Gamer (UK) (Vol. 01 Nº 03 - February 1994) [MarkHnsn, archive.org]
PC Gamer (UK) (Vol. 01 Nº 04 - March 1994) [MarkHnsn, archive.org]
PC Gamer (UK) (Vol. 01 Nº 05 - April 1994) [MarkHnsn, archive.org]
PC Gamer (UK) (Vol. 01 Nº 07 - June 1994) [MarkHnsn, archive.org]
PC Gamer (UK) (Vol. 02 Nº 10 - September 1995) [SmartCoda, archive.org]
PC Gamer (USA) (Vol. 01 Nº 01 - May-June 1994) [OUTATIME Studios, archive.org]
PC Gamer (USA) (Vol. 01 Nº 03 - August 1994) [OUTATIME Studios, archive.org]
PC Gamer (USA) (Vol. 01 Nº 04 - September 1994) [OUTATIME Studios, archive.org]
PC Gamer (USA) (Vol. 01 Nº 06 - November 1994) [OUTATIME Studios, archive.org]
PC Gamer (USA) (Vol. 02 Nº 03 - March 1995) [MarkHnsn, archive.org]
PC Review (Issue 07 - May 1992) [archive.org]
PC Review (Issue 25 - November 1993) [MarkHnsn, archive.org]

New NOT working software list additions
-----------------------------------
PC Gamer (UK) (Vol. 01 Nº 02 - January 1994) [MarkHnsn, archive.org]

### CODE REVIEW

File: hash/ibm5170.xml [hunk 1]
```diff
@@ -21838,6 +22156,34 @@ license:CC0
...
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="pcr199311">
+		<description>PC Review (Issue 25 - November 1993)</description>
+		<year>1993</year>
+		<publisher>EMAP Images</publisher>
+		<notes>Disk includes: "Ancients: Death Watch (demo)"</notes>
+		<!-- Missing Disk A -->
```
Comment (cuavas) [inline hash/ibm5170.xml:22178]: Should you add a `NO_DUMP` part for the missing disk?  It’s almost invisible like this.

**Review by cuavas — CHANGES_REQUESTED** [review]
Please don’t use an N and a degree symbol or masculine ordinal for the “number” abbreviation – use the correct character № (U+2116 Numero Sign).

File: hash/ibm5170.xml (hunk 1 above)
Comment (ArcadeShadow) [reply hash/ibm5170.xml]: Excellent idea! Corrected.

**Comment by ArcadeShadow** [comment]
I didn't know about this Numero Sign. 
I replaced by that symbol.
Please check if it's OK.

**Comment by Tafoid** [comment]
Looks ok to me.  We can always revisit this minor label issue later if needed.

**Pull request merged**